python/modules/IcePy/Types.cpp

```diff
@@ -1772,7 +1772,7 @@ IcePy::SequenceInfo::marshalPrimitiveSequence(const PrimitiveInfoPtr& pi, PyObje
 
             if(pi->kind != PrimitiveInfo::KindByte)
             {
-#   ifdef ICE_BIG_ENDIAN
+#   ifdef WORDS_BIGENDIAN
```

Comment:
What is this macro?

Reply:
It's a C-Python macro, set to 1 on big endian systems.